docs/satellite-repos.md

```diff
@@ -0,0 +1,17 @@
+# Satellite repos
+
+We have seven satellite repos that are maintained outside of this codebase.
+
+The satellite apps are:
+
+- [ccdb5_api](https://github.com/cfpb/ccdb5-api)
+- [ccdb5_ui](https://github.com/cfpb/ccdb5-ui)
+- [college_costs](https://github.com/cfpb/college-costs)
+- [comparisontool](https://github.com/cfpb/django-college-costs-comparison)
+- [owning_a_home_api](https://github.com/cfpb/owning-a-home-api)
+- [retirement](https://github.com/cfpb/retirement)
+- [teachers_digital_platform](https://github.com/cfpb/teachers-digital-platform)
```

Comment:
Can we add a line here about how we have a separate requirements file, [requirements/optional-public.txt](https://github.com/cfpb/cfgov-refresh/blob/master/requirements/optional-public.txt), that specifies wheel files for these?